sfaira/data/dataloaders/loaders/d10_1038_nature22330/homosapiens_forebrain_2017_microwellseq_birey_001.yaml

```diff
@@ -0,0 +1,89 @@
+dataset_structure:
+    dataset_index: 1
+    sample_fns:
+dataset_wise:
+    author: "Birey, Fikri"
+    default_embedding: 
+    doi_preprint: 
+    doi_journal: "10.1038/nature22330"
+    download_url_data: "https://www.ncbi.nlm.nih.gov/geo/download/?acc=GSE93811&format=file"
+    download_url_meta: 
+    primary_data: True
+    year: 2017
+layers:
+    layer_counts: "X"
+    layer_processed: 
+    layer_spliced_counts: 
+    layer_spliced_processed: 
+    layer_unspliced_counts: 
+    layer_unspliced_processed: 
+    layer_velocity: 
+dataset_or_feature_wise:
+    feature_reference: 
+    feature_reference_var_key: 
+    feature_type: "rna"
+    feature_type_var_key: 
+dataset_or_observation_wise:
+    assay_sc: "microwell-seq"
+    assay_sc_obs_key: 
+    assay_differentiation: "Pasca, 2015" 
+    assay_differentiation_obs_key: 
+    assay_type_differentiation: "guided"
+    assay_type_differentiation_obs_key: 
+    bio_sample:
+    bio_sample_obs_key: 
+    cell_line: 
+    cell_line_obs_key: 
```

Comment:
I would say the cell line information is quite crucial here. it seems really hard to identify from the paper though. can you try to dig a little to see if we can somehow define the cell_line that was used for these organoids? (supplementary table 1 states that both the lines 2242-1 and 8858-1 were used for the single-cell. but it's not quite clear to me where each of these was used.) if you cannot find the information you can maybe email the corresponding author and ask them.